src/addons/exporter/exports/README

```diff
@@ -0,0 +1,6 @@
+When executing a data export using the exporter Addon,
```

Comment:
As mentioned in the other PR you can name it `.md` and put markdown, and it will still be "human readable" even looking at the raw text.  And the web version in github will have a styled view using markdown.

Reply:
Beat you this time.